### ISSUE TITLE
Bus Error: 10

### ISSUE THREAD

**Issue by Feliix42** [centaurean/sharc#21]
Hey, 
I cloned this repo to my MacBook Air (Mid 2013, 1.7 GHz i7, 8GB RAM), running OS X 10.10.2 Yosemite and built sharc. I was able to compress a PDF-file without problems at default compression level. As I tried to run `./sharc -x -c3 git-folien2.pdf`I received *Bus Error:10*.

gdb says 
```
(gdb) run
Starting program: /Users/Felix/GitHub/sharc/sharc -x -c3 git-folien2.pdf
warning: Could not open OSO archive file "/BinaryCache/corecrypto/corecrypto-233.1.2~26/Symbols/BuiltProducts/libcorecrypto_static.a"
warning: `/BinaryCache/coreTLS/coreTLS-35.10.1~5/Objects/coretls.build/coretls.build/Objects-normal/x86_64/system_coretls_vers.o': can't open to read symbols: No such file or directory.
warning: Could not open OSO archive file "/BinaryCache/coreTLS/coreTLS-35.10.1~5/Symbols/BuiltProducts/libcoretls_ciphersuites.a"
warning: Could not open OSO archive file "/BinaryCache/coreTLS/coreTLS-35.10.1~5/Symbols/BuiltProducts/libcoretls_handshake.a"
warning: Could not open OSO archive file "/BinaryCache/coreTLS/coreTLS-35.10.1~5/Symbols/BuiltProducts/libcoretls_record.a"
warning: Could not open OSO archive file "/BinaryCache/coreTLS/coreTLS-35.10.1~5/Symbols/BuiltProducts/libcoretls_stream_parser.a"

Program received signal SIGBUS, Bus error.
0x0000000100052236 in density_lion_encode_continue ()
```

I would be happy if you could enlighten me. Is it a bug or did I miss something? :)

**Comment by gpnuma** [comment]
Hello,
Yes it's a bug, it's the same as #20. The issue is lion specific (-c3).
Thanks for your input.

**Comment by gpnuma** [comment]
Fixed in 78128cbce84bbeef1c8f1f879f00bc4a6cb82527
